CGCharter-1727386911.html

```diff
@@ -41,17 +41,18 @@ <h1>
       </li>
       <li>Previous Charter: <span class="remove">{TBD: URI}</span>
       </li>
-      <li>Start Date: <span class="remove">{TBD: date the charter takes effect,
-      estimate if not known. Update this if the charter is revised and include
-      a link to the previous version of the charter.}</span>
+      <li>Start Date: <span class="remove">1 January 2025</span>
```

Comment:
```suggestion
      <li>Start Date: <span class="remove">6 January 2025</span>
```

Assuming approval at today's meeting.